docker-compose.yml

```diff
@@ -1,16 +1,16 @@
 version: '2'
 
 services:
-    dockerhost:
-        image: qoomon/docker-host
-        cap_add: [ 'NET_ADMIN', 'NET_RAW' ]
-        mem_limit: 8M
-        restart: on-failure
-    tcp_message_emitter:
-        depends_on: [ dockerhost ]
-        image: alpine
-        command: [ "sh", "-c", "while :; do echo -n 'TCP - '; date; sleep 1; done | nc 'dockerhost' 2323 -v"]
-    udp_message_emitter:
-        depends_on: [ dockerhost ]
-        image: alpine
-        command: [ "sh", "-c", "while :; do echo -n 'UDP - '; date; sleep 1; done | nc 'dockerhost' 5353 -u -v"]
+  dockerhost:
```

Comment:
Is there any diff except indentation? If that's the only change please revert it.

Reply:
Yes, that commit did only formatting indeed. I did that under the assumption that YAML is always indented using 2 spaces, as I have simply never seen otherwise. Turns out that I was wrong, and the YAML spec doesn't mandate any indentation style, as long as it's done using spaces and the next indentation level has at least 1 more space than the previous level. So I will revert this back.

Reply:
Thanks, don't get me wrong I'm totally fine with 2 space indentation, however I'd like to keep this PR clean to the podman purpose. Feel free to create another reformat PR.

Reply:
Agreed! The indentation does not bother me, it just felt a bit weird or even wrong seeing that, so I changed it. I also doubted if I shouldn't make indeed another reformat PR, but it felt like such a tiny thing that I could just include it with this one. But I have no problem keeping it with 4-spaces :)